client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-content/style.scss

```diff
@@ -4,6 +4,7 @@
 	padding: 48px;
 	text-align: center;
 	overflow-y: auto;
+	height: calc(100vh - 265px);
```

Comment:
Can you leave a comment explaining why 165px? If we need to change this in the future, at least we will know how to calculate the offset again.

Reply:
It's because of the header height. It calculates the parent height (the viewport) and subtracts the header size to determine the height of the scrollable view.